javascript/javascript2/week1/homework/yourSpritAnimal/yourSpiritAnimal.js

```diff
@@ -0,0 +1,54 @@
+//Spirit animal name generator
+
+const spiritAnimal = ["The curious cat", "The confident deer", "The courageous lion", "The fullmoon wolf", "The loyal dog", "The wise owl", "The compassionate hawk", "The intelligent fox", "The crying butterfly", "The quick rabbit", ]
```

Comment:
Longs arrays are easier to read formatted like this.
```suggestion
const spiritAnimal = [
    "The curious cat",
    "The confident deer",
    "The courageous lion",
    "The fullmoon wolf",
    "The loyal dog",
    "The wise owl",
    "The compassionate hawk",
    "The intelligent fox",
    "The crying butterfly",
    "The quick rabbit", 
    ]
```